<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>10</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="username_label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="password_label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_inpt">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;;border-radius: 3px</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pass_inpt">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;;border-radius: 3px</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login_btn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="not_authorized">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color:blue</string>
      </property>
      <property name="text">
       <string>Login Page</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="from_label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>230</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>From</string>
      </property>
     </widget>
     <widget class="QLabel" name="to_label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>280</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>To</string>
      </property>
     </widget>
     <widget class="QComboBox" name="from_combo">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>230</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>USD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GEL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EUR</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="to_combo">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>280</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>GEL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EUR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>USD</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="money_inpt">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>230</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;;border-radius: 3px</string>
      </property>
     </widget>
     <widget class="QPushButton" name="convert_btn">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>350</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_btn">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>350</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QLabel" name="converted_money">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>270</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:green</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="logout_btn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>450</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QLabel" name="title_exchangeapp">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#00008B</string>
      </property>
      <property name="text">
       <string>Currency Converter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>180</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
